src/main/java/org/osc/manager/rest/server/api/PolicyApis.java

```diff
@@ -0,0 +1,115 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.manager.rest.server.api;
+import java.util.List;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.apache.log4j.Logger;
+import org.osc.manager.ism.entities.PolicyEntity;
+import org.osc.manager.rest.server.SecurityManagerServerRestConstants;
+import org.osgi.service.component.annotations.Component;
+
+@Component(service = PolicyApis.class)
+@Path(SecurityManagerServerRestConstants.SERVER_API_PATH_PREFIX + "/policy")
```

Comment:
Looks like this url will end up being /policy/domains/domainid/{did}/policies/{policyid}, it should be just /domains/{did}/policies/{policyid}, without the initial  '/policy/'.  For that you may need to change the route here to be actually **/domains**. Observe that it will be the same route as the DomainApis when you have that in the future , which means at least two classes DomainApis and PolicyApis will have the same route, if that is not possible you can have a single class DomainApis routed to /domains and put the policy methods in it. On the previous example I mentioned this was the approach, the VirtualizationConnectorApis have the security group methods and they are routed under url /virtualizationconnectors/